src/api/app/controllers/source_project_config_controller.rb

```diff
@@ -7,15 +7,7 @@ def show
 
     sliced_params = slice_and_permit(params, [:rev])
 
-    return if forward_from_backend(config.full_path(sliced_params))
-
-    content = config.content(sliced_params)
-
-    unless content
-      render_404(config.errors.full_messages.to_sentence)
-      return
-    end
-    send_config(content, config.response[:type])
+    pass_to_backend(config.full_path(sliced_params))
```

Comment:
There is one difference here. `content` was solely in memory, `pass_to_backend` stores it in a tmpfile. I don't think this makes any difference to this route.